pkg/kmm/kubelet.go

```diff
@@ -17,15 +17,26 @@ import (
 // CreateAndStartKubelet will call the CreateAndStartKubelet method with the correct configuration
 func (k *Kmm) CreateAndStartKubelet(master bool) error {
 
+	nodeLables := ""
+	if k.NodeLabels != nil {
+		for k, v := range k.NodeLabels {
+			if len(nodeLables) > 0 {
+				nodeLables = nodeLables + ","
```

Comment:
would this not place a ',' at the end of the line? .. perhaps just add a list a strings.Join with a comma